src/components/BaseInput.vue

```diff
@@ -32,7 +32,10 @@ export default {
       type: String,
       default: "",
     },
+   
   },
+
```

Comment:
Such empty lines should generally be removed. If you'd have eslint installed this would throw a warning 😄